libusb/descriptor.c

```diff
@@ -1070,6 +1113,27 @@ void API_EXPORTED libusb_free_container_id_descriptor(
 	free(container_id);
 }
 
+int usbi_string_descriptor_to_ascii(union usbi_string_desc_buf* str,
```

Comment:
I would think to utf8 would be more useful.

Reply:
I believe this function is extracted from existing code and re-used

but in general - is there a UTF16->UTF8 mechanism library already present in libusb (as a dependency)?

---
generally I agree, people rarely need pure ascii nowadays